src/components/ButtonPanel.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable arrow-body-style */
+import React from 'react';
+import Button from './Button';
+
+const ButtonPanel = () => {
+  return (
+    <div className="calculator__panel">
+      <div id="group-1" className="calculator__panel--buttonsRow">
+        <Button buttonName="AC" color />
+        <Button buttonName="+/-" color />
+        <Button buttonName="%" color />
+        <Button buttonName="/" />
+      </div>
+
+      <div id="group-2" className="calculator__panel--buttonsRow">
+        <Button buttonName="7" color />
```

Comment:
[Optional]

We could rewrite the ButtonPanel component to use two loops to make the `Button` instead of hardcoding them. By doing this, we keep our code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

We could achieve this by creating an array of symbol and loop through the array using [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.